pkg/users/account-create-dialog.js

```diff
@@ -112,7 +112,7 @@ function AccountCreateBody({ state, errors, change, shells }) {
             <FormGroup label={_("Authentication")} fieldId="accounts-create-locked" hasNoPaddingTop>
                 <Radio id="account-use-password"
                        label={_("Use password")}
-                       isChecked={!locked} onChange={checked => change("locked", !checked)}
+                       isChecked={!locked} onChange={(_, checked) => change("locked", !checked)}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-135116-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)